notes.md

```diff
@@ -0,0 +1,79 @@
+## Architecture change suggestions
+- should use consistent login credentials to cs playground
+- host separator should be "-" like the default since this is what customers will do
+- should use oathbearer for client -> gw since this is what customers will do
+- should use gateway provider for kafkacluster if we are going to use console to connect to gateway at all
+- should also include a kafkacluster that bypasses gateway
+
+retrieve truststore for local kafka client
+```
+kubectl get secret bundle-truststore -n conduktor -o jsonpath='{.data.truststore\.jks}' | base64 --decode > truststore.jks
+```
+
+```
+export KAFKA_OPTS="-Djava.security.manager=allow"
+```
+
+create client properties
+```
+security.protocol=SASL_SSL
+sasl.mechanism=PLAIN
+sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username='console-sa' password='<token>';
+ssl.truststore.location=./truststore.jks
+ssl.truststore.password=conduktor
+```
+
+Run kafka client from outside world
+```
+kafka-broker-api-versions \                      
+    --bootstrap-server gateway.conduktor.localhost:9092 \
+    --command-config client.properties | grep 9092
+```
+
+Port forward grafana to take a look at the dashboards, or alternatively expose with ingress
+
+```
+kubectl port-forward svc/grafana-service -n monitoring 3000:3000
+```
+Log in with `admin` and `admin` for username, password.
+
+## installation issues
+
+- need kubectl 1.33+
+- need helm repo update on `make install-conduktor-platform`
+- timeouts not long enough
+
+## console tls issues
+```
+Error: execution error at (console/templates/console/ingress.yaml:58:14): Ingress TLS enabled require one of : 
```

Comment:
Indeed, I think it's something we push recently, and for passthrough to work we need to disable tls on ingress and just add corresponding annotations.

Like this 
```yaml
ingress:
  enabled: true
  ingressClassName: nginx
  hostname: console.conduktor.localhost
  tls: false # because we use SSL passthrough
  annotations:
    # SSL passthrough for TLS termination on the Console container
    nginx.ingress.kubernetes.io/ssl-passthrough: "true"
    nginx.ingress.kubernetes.io/ssl-redirect: "false"
    ingress.kubernetes.io/ssl-passthrough: "true"
    nginx.ingress.kubernetes.io/backend-protocol: HTTPS
```